Agreements/Internal/27_TCCOW.txt

```diff
@@ -0,0 +1,17 @@
+COPX:*:*:TOBID:*:EGLL:London S27:London TC COWLY:*:20000:TOBID
+COPX:*:*:TOBID:*:EGWU:London S27:London TC COWLY:*:20000:TOBID
+
+COPX:*:*:DISIT:*:EGKK:London S27:London TC COWLY:*:20000:DISIT
```

Comment:
I think it might be worth a catch all here, for those aircraft via the L612 in case their cruise level is high or something. And also I quite often see aircraft without the N859 portion along this segment

```suggestion
COPX:*:*:DISIT:*:EGKK:London S27:London TC COWLY:*:20000:DISIT
COPX:*:*:*:*:EGKK:London S27:London TC COWLY:*:20000:DISIT
```

Reply:
Suggestion not accepted? :)

Reply:
Apparently some issue pushing my commits. It's here https://github.com/VATSIM-UK/UK-Sector-File/pull/2124/commits/f524ae3f3fefa2d773d25973e7e76ae2014821a7#diff-1c0da162a78f004c06b1cb4a2dbc0477R5